manifests/init.pp

```diff
@@ -137,8 +137,8 @@
       }
     }
     'Suse': {
```

Comment:
This has bit us before. Would you mind putting a comment here explaining the lack of the fact and why we are using the regex? Else I'm afraid someone will send a patch to revert it and it will get merged or worse I'll be afraid to merge anything related to this with Suse.. which has also happened elsewhere